packages/pageflow/src/frontend/videoPlayer/Lazy.js

```diff
@@ -1,8 +1,12 @@
 import $ from 'jquery';
 import _ from 'underscore';
+import {VideoPlayer} from './VideoPlayer';
+import {browser} from '../browser';
+import {log} from '../base';
+import {state as pageflow} from '../state';
```

Comment:
These alias imports hide errors. In this file there are instances of `pageflow.browser` that have been forgotten due to this. It works since since it implicitly relies on the global, but there actually is no `state` here.